mapie/regression/time_series_regression.py

```diff
@@ -367,8 +367,9 @@ def predict(
         self.lower_quantiles_ = lower_quantiles
         self.higher_quantiles_ = higher_quantiles
 
-        if self.method in self.no_agg_methods_ \
-                or self.cv in self.no_agg_cv_:
+        if self.method in self.no_agg_methods_ or (
```

Comment:
The brackets here are strange. Was this done on purpose?

Reply:
I could change this.